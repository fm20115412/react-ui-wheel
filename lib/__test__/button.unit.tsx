import rerender from 'react-test-renderer'
import Button from '../button'
import React from 'react'

describe('button', () => {
    it('这是个button', () => {
        let json = rerender.create(<Button />).toJSON()
        expect(json).toMatchSnapshot()
    })
})