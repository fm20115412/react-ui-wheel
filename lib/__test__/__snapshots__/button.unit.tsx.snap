// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button 这是个button 1`] = `
<button>
  click me
</button>
`;
